code/controllers/configuration.dm

```diff
@@ -273,6 +273,8 @@ var/list/gamemode_cache = list()
 
 	var/merchant_chance = 20 //Chance, in percentage, of the merchant job slot being open at round start
 
+	var/show_game_type_odd = 1 // If the game type odds are shown in server revision

```

Comment:
The description is misleading.
That var is used to change if the verb can be used or not